app/build.gradle

```diff
@@ -94,11 +100,11 @@ dependencies {
 
     //Dagger Hilt
     implementation "com.google.dagger:hilt-android:$versions.hilt"
-//    implementation project(path: ':shcDecoder')
+    implementation project(path: ':shcDecoder')
```

Comment:
I'd just get rid of the commented out bits

Reply:
Yes, I will do this clean up once we have require things in place